scripts/generate_linux_tests.rb

```diff
@@ -1,241 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
So we'll still need it because 5.0

Reply:
How about 5.2, do we still need it there? IIRC the test discoverability flag got deprecated only in 5.3, so we'd probably still need it.